src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -132,10 +135,10 @@ Agent::VelocityObstacle HRVOAgent::createVelocityObstacle(const Agent &other_age
             0.5f * (other_agent.getVelocity() + velocity_) -
             (uncertaintyOffset_ + 0.5f *
                                       (other_agent.getRadius() + radius_ -
-                                       abs(position_ - other_agent.getPosition())) /
+                                       (position_ - other_agent.getPosition()).length()) /
                                       simulator_->timeStep_) *
-                normalize(position_ - other_agent.getPosition());
-        velocityObstacle.side1_ = normal(other_agent.getPosition(), position_);
+                (position_ - other_agent.getPosition()).normalize();
+        velocityObstacle.side1_ = (other_agent.getPosition(), position_).perpendicular();
```

Comment:
Not fully sure if the `perpendicular` call here is doing the same thing as `normal`. Based on `normal`'s documentation, it takes two vectors (points) which make a line, but `perpendicular` take a single vector (`this` object), and assumes that it is the vector/line you want to make perpendicular. My guess is that it should be something like:
```cpp
(position_ - other_agent.getPosition()).perpendicular()
```
But I'm not fully sure if the order of the subtraction is correct. 
If you make changes, you should probably run the tests again to make sure that it's still working as expected.